pkg/network/kuryr.go

```diff
@@ -196,6 +204,12 @@ func validateKuryr(conf *operv1.NetworkSpec) []error {
 		}
 	}
 
+	if kc != nil {
+		if kc.MTU != nil && (*kc.MTU < 576 || *kc.MTU > 65536) {
```

Comment:
We don't support `0` anymore? Then check on line 117 is not required, right?

Reply:
right